src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTargetScaler.cs

```diff
@@ -57,7 +57,16 @@ internal TargetScalerResult GetScaleResultInternal(TargetScalerContext context,
                 concurrency = DefaultMaxItemsPerInvocation;
             }
 
-            int targetWorkerCount = (int)Math.Ceiling(remainingWork / (decimal)concurrency);
+            int targetWorkerCount;
+
+            try
+            {
+                targetWorkerCount = (int)Math.Ceiling(remainingWork / (decimal)concurrency);
```

Comment:
I suggest wrapping in a checked block - this overwrites the project level overflow setting.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/checked-and-unchecked

Reply:
done, updated all other SDK PRs to match this implementation